client/src/theme/Components/browse/event/_events.scss

```diff
@@ -126,6 +126,10 @@
       padding-left: 36px;
     }
 
+    > div {
```

Comment:
Good call. In general, we don't want to use selectors as unspecific as `> div` unless it's for a `ul` inside a `nav` or something (because that's a very common case) but it works well here, since it's a simple flex item wrapper.